docs/src/verification.md

```diff
@@ -0,0 +1,164 @@
+
+So generally we want to perform the following steps:
+1. Create a framework model
+2. Run a forward pass through the framework model
+3. Compile the framework model using `Forge`
+4. Run a forward pass through the compiled model
+5. Compare the outputs.
+
+Most of the above steps `verify()` function does for us:
+- Handles forward passes for both framework and compiled models
+- Compares results using a combination of comparison methods
+- Supports customization through the `VerifyConfig` class.
+
+**Example of usage**
+```python
+def test_add():
+
+	class Add(nn.Module):
+		def __init__(self):
+			super().__init__()
+
+		def forward(self, a, b):
+			return a + b
+
+
+	inputs = [torch.rand(2, 32, 32), torch.rand(2, 32, 32)]
+
+	framework_model = Add()
+	compiled_model = forge.compile(framework_model, sample_inputs=inputs)
+
+	verify(inputs, framework_model, compiled_model)
+```
+
+**Notes:**
+- If you only want to compile model and perform forward pass without comparing outputs you can just:
+```python
+def test_add():
+
+	class Add(nn.Module):
+		def __init__(self):
+			super().__init__()
+
+		def forward(self, a, b):
+			return a + b
+
+
+	inputs = [torch.rand(2, 32, 32), torch.rand(2, 32, 32)]
+
+	framework_model = Add()
+	compiled_model = forge.compile(framework_model, sample_inputs=inputs)
+
+	fw_out = framework_model(*inputs)
+	co_out = compiled_model(*inputs)
+```
+
+## Verify Config Overview
+
+If `VerifyConfig` isn't passed as a param, default one will be used. Currently through `VerifyConfig` you can disable/enable:
+| Feature                           | Enabled (default) |
+|-----------------------------------|:-----------------:|
+| Verification as a method          | `True`            |
+| Output size check                 | `True`            |
+| Output type check                 | `True`            |
+| Output shape check                | `True`            |
+
+For more information about `VerifyConfig` you can check `forge/forge/verify/config.py`.
+
+**Example of usage**
+```python
+def test_add():
+
+	class Add(nn.Module):
+		def __init__(self):
+			super().__init__()
+
+		def forward(self, a, b):
+			return a + b
+
+
+	inputs = [torch.rand(2, 32, 32), torch.rand(2, 32, 32)]
+
+	framework_model = Add()
+	compiled_model = forge.compile(framework_model, sample_inputs=inputs)
+
+	fw_out = framework_model(*inputs)
+	co_out = compiled_model(*inputs)
+
+    verify(inputs, framework_model, compiled_model, VerifyConfig(verify_dtype=False))
+```
+</br>
+
+Besides that, config also includes value checker. There are 3 types of checker:
+- `AutomaticValueChecker` **(default)**
+- `AllCloseValueChecker`
+- `FullValueChecker`
+
+For more information about **Checkers** you can look at `forge/forge/verify/value_checkers.py`.
+
+<br/>
+<br/>
+
+<img src="imgs/checkers/checkers.svg" alt="Checkers" style="width: 1200%;">
```

Comment:
let's see what others say, but to me the picture doesn't add much value and makes the doc harder to edit/update

Reply:
@nvukobratTT what do you think?

Reply:
I'm okay with using images and they can be really useful as part of documentaion. However, it's important that those can be edit/updated over time. 

That said, I would opt out for using ExcallyDraw to create this and similar diagrams, and next to the image also store exported ED file which we can load and edit in the future.